src/components/component-playground.less

```diff
@@ -264,6 +236,112 @@
                            @checkerboard-square-size @checkerboard-square-size;
     }
 
+    > .split-pane {
+
+      > .pane {
+
+        > .fixture-editor-outer {
+          height: 100%;
+          width: 100%;
+          border: 1px solid @editor-border-color;
+          background: #f8f5ec;
+
+          > .fixture-editor {
+            width: 100%;
+            height: 100%;
+            padding: @default-spacing;
+            box-sizing: border-box;
+            border: none;
+            background: transparent;
+            color: #637c84;
+            font-family: @editor-font-family;
+            font-size: @editor-font-size;
+            line-height: @editor-line-height;
+            resize: none;
+            outline: none;
+
+            &.invalid-syntax {
+              color: #cc7a6f;
+            }
+          }
+        }
+
+        > .preview {
+          height: 100%;
+          overflow: auto;
+          // Checkerboard effect on background
+          background-image:
+            linear-gradient(45deg,
+                            @checkerboard-color 25%,
+                            transparent 25%,
+                            transparent 75%,
+                            @checkerboard-color 75%,
+                            @checkerboard-color 100%),
+            linear-gradient(45deg,
+                            @checkerboard-color 25%,
+                            transparent 25%,
+                            transparent 75%,
+                            @checkerboard-color 75%,
+                            @checkerboard-color 100%);
+          background-size: 2 * @checkerboard-square-size
+                           2 * @checkerboard-square-size;
+          background-position: 0 0,
+                               @checkerboard-square-size @checkerboard-square-size;
+        }
+      }
+
+      > .resizer {
+        background: #000;
+        opacity: .2;
+        z-index: 1;
+        -moz-box-sizing: border-box;
+        -webkit-box-sizing: border-box;
+        box-sizing: border-box;
+        -moz-background-clip: padding;
+        -webkit-background-clip: padding;
+        background-clip: padding-box;
+
+        &:hover {
```

Comment:
Discussed in person to remove this because was causing weird UI problems.
![wololo](https://cloud.githubusercontent.com/assets/13434505/16990849/4212faaa-4ea2-11e6-976c-8d69bd8b850c.gif)